src/homeWork/module8/task1/CompressorForWords.java

```diff
@@ -0,0 +1,70 @@
+package homeWork.module8.task1;
+
+/**
+ * Created by : Volodymyr_Silitskyi
+ * Created at : 6/4/2018
+ */
+
+
+public class CompressorForWords {
+    public static void main(String[] args) {
+        String stringForCompress = "h2elloooooooooooo2";
+        String stringForUncompress = "hel2o9o9o9o9o2";
+
+        compressString(stringForCompress);
+        uncompressString(stringForUncompress);
+    }
+
+    public static void compressString(String stringForCompress) {
```

Comment:
return `String` rather then `void`. It will be better design to explicitly call `System.out.println(compressString(stringForCompress))`